tests/integrations/config/mod.rs

```diff
@@ -227,7 +227,8 @@ fn test_serde_custom_tikv_config() {
         writable_file_max_buffer_size: ReadableSize::mb(12),
         use_direct_io_for_flush_and_compaction: true,
         enable_pipelined_write: false,
-        enable_unordered_write: true,
+        enable_multi_batch_write: true,
+        enable_unordered_write: false,
```

Comment:
This is supposed to be different from its default value.

Reply:
ok